dh2000Sup/dh2000.db

```diff
@@ -0,0 +1,66 @@
+record(bo, "$(P)SIM")
+{
+    field(SCAN, "Passive")
+    field(DTYP, "Soft Channel")
+    field(ZNAM, "NO")
+    field(ONAM, "YES")
+    field(VAL, "$(RECSIM=0)")
+    field(PINI, "YES")
+}
+
+record(bo, "$(P)DISABLE")
+{
+    field(DESC, "Disable comms")
+    field(PINI, "YES")
+    field(VAL, "$(DISABLE=0)")
+    field(OMSL, "supervisory")
+    field(ZNAM, "COMMS ENABLED")
+    field(ONAM, "COMMS DISABLED")
+}
+
+record(bi, "$(P)INTERLOCK"){
+    field(DESC, "Status of the interlock")
+    field(SCAN, "Passive")
+    field(ZNAM, "OKAY")
```

Comment:
Standard is to use `OK` rather than `OKAY` (see https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/PV-Naming#standard-signal-names )